pydicom/tests/test_uid.py

```diff
@@ -1,93 +1,22 @@
-# test_uid.py
 """Test suite for uid.py"""
-# Copyright (c) 2008-2012 Darcy Mason
-# This file is part of pydicom, released under a modified MIT license.
-#    See the file LICENSE included with this distribution, also
-#    available at https://github.com/pydicom/pydicom
-
```

Comment:
What happened to the copyright? I'm not sure what the general policy on this is here, seems to be sometimes present, sometimes not (@darcymason?)